bin/inference/pycbc_inference_start_from_dynesty

```diff
@@ -0,0 +1,46 @@
+#!/bin/env python
+""" Convert dynesty output file to emcee_pt compatible start format
+"""
+
+import argparse
+import numpy
+from numpy.random import choice
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--input-file')
+parser.add_argument('--output-file')
+parser.add_argument('--ntemps', type=int)
+parser.add_argument('--nwalkers', type=int)
+args = parser.parse_args()
+
+# populate an emcee start file with
+# values chosen from a dynesty file
+# each temperature and walker will get a random
+# point from the dynesty output
+
+ntemps = args.ntemps
+nwalkers = args.nwalkers
+
+import h5py
+f = h5py.File(args.input_file, 'r')
+
+nsample = len(f['samples'][f['samples'].keys()[0]])
```

Comment:
Yeah, I agree. Hence my top-level comment above. I realized this would be a better way to go and would be needed anyway once the dyensty format changes.